### ISSUE TITLE
Property 'un' does not exist on type 'LocalNotifications'

### ISSUE THREAD

**Issue by ashwin-kambli** [katzer/cordova-plugin-local-notifications#1585]
I upgraded my plugin from 4.3.2 to 4.7.0. Now, I get an error: "Property 'un' does not exist on type 'LocalNotifications'." 
Why was the "un" function removed in the new version?  

## Your Environment

* Plugin version: 4.7.0
* Platform: Android
* OS version: Windows 7
* Device manufacturer / model: Samsung Galaxy S7
* Cordova version (```cordova -v```): 6.5.0
* Cordova platform version (```cordova platform ls```): Android 5.2.2
* Plugin config
* Ionic Version (if using Ionic): 2.3.0

## Expected Behavior
I want to be able to un-register from the "click" event by calling "un".

## Actual Behavior
"un" is no longer a property on LocalNotifications

**Comment by rwillett** [comment]
What plugin are you using? The plugin versions for here are 0.9.x. 

What plugin has version 4.3.2? Is this the Ionic version of this plugin?

**Comment by ashwin-kambli** [comment]
Yes, this is the Ionic version. Here is an excerpt from my package.json: 
`     "dependencies": {`
`        "@angular/common": "2.4.8",`
`        "@angular/compiler": "2.4.8",`
`        "@angular/compiler-cli": "2.4.8",`
`        "@angular/core": "2.4.8",`
`        "@angular/forms": "2.4.8",`
`        "@angular/http": "2.4.8",`
`        "@angular/platform-browser": "2.4.8",`
`        "@angular/platform-browser-dynamic": "2.4.8",`
`        "@angular/platform-server": "2.4.8",`
`        "@angular/tsc-wrapped": "4.4.3",`
`        "@ionic-native/core": "4.3.2",`
`        "@ionic-native/local-notifications": "4.7.0",`
